Source/WebCore/style/StyleBuilderCustom.h

```diff
@@ -208,13 +208,6 @@ inline void BuilderCustom::applyValueZoom(BuilderState& builderState, CSSValue&
     if (primitiveValue.valueID() == CSSValueNormal) {
         resetUsedZoom(builderState);
         builderState.setZoom(RenderStyle::initialZoom());
-    } else if (primitiveValue.valueID() == CSSValueReset) {
-        builderState.setUsedZoom(RenderStyle::initialZoom());
-        builderState.setZoom(RenderStyle::initialZoom());
-    } else if (primitiveValue.valueID() == CSSValueDocument) {
```

Comment:
You can remove `document` from [CSSValueKeywords.in](https://searchfox.org/wubkat/source/Source/WebCore/css/CSSValueKeywords.in) because it's not used anywhere else.

Reply:
@nt1m - done. :-)